VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -196,7 +193,9 @@
                         android:layout_height="@dimen/course_unit_nav_bar_btn_height"
                         android:background="@color/white"
                         android:text="@string/assessment_previous"
-                        android:textColor="?attr/buttonTextColor" />
+                        android:textColor="?attr/buttonTextColor"
+                        android:gravity="center_vertical|start"
```

Comment:
Nope, this is required. We need the text of the button to be left aligned which causes it to move up, requiring us to add center_vertical with it.